examples/gotutorial.md

```diff
@@ -28,7 +28,7 @@ Then change your current directory to `grpc-go/examples/route_guide`:
 $ cd $GOPATH/src/google.golang.org/grpc/examples/route_guide
 ```
 
-You also should have the relevant tools installed to generate the server and client interface code - if you don't already, follow the setup instructions in [the Go quick start guide](https://github.com/grpc/grpc-go/tree/master/examples/).
+You also should have the relevant tools installed to generate the server and client interface code - if you don't already, follow the setup instructions in [the Go quick start guide](https://grpc.io/docs/languages/go/quickstart).
```

Comment:
Just following the other PR's review, shall we change this to something like
```
"Ensure you have the relevant tools installed to generate the server and client interface code. If you don't, follow the setup instructions in the Go quick start guide (https://grpc.io/docs/languages/go/quickstart)."
```
What do you think @arjan-bal?

Reply:
Yeah, it sounds better. Changed accordingly.